pages/Telemetry.md

```diff
@@ -38,6 +38,23 @@ Redix connections (both `Redix` and `Redix.PubSub`) execute the following Teleme
     * `:address` - the address the connection successfully reconnected to.
     * `:conn` - the PID of the Redix connection that emitted the event.
 
+`Redix` connections execute the following Telemetry events when commands or pipelines of any kind are executed.
+
+  * `[:redix, :pipeline]` - executed when a pipeline (or command, which is a pipeline with just one command) is successfully sent to the server and a reply comes from the server. Measurements are:
+
+    * `:elapsed_time` (integer) - the elapsed time in microseconds that it took to send the pipeline to the server and get a reply.
```

Comment:
I know `microseconds` is the unit that `:timer.tc` returns, but other recent instrumentations have leveraged the `:native` time unit via `System.monotonic_time()`.

There isn't a defined standard, but perhaps we should consider making recommendations for time units across `:telemetry`?

* Plug: https://github.com/elixir-plug/plug/blob/master/lib/plug/telemetry.ex#L30-L33
* Absinthe: https://github.com/absinthe-graphql/absinthe/pull/663/files#diff-48f3714903e04ae243798591708567d3R31